docs/content/configuration/creating_volumestorageclass.md

```diff
@@ -24,14 +24,16 @@ Use the `SpaceEfficiency` parameters for each storage system, as defined in [the
 
 - The IBM DS8000 family `pool` value is the pool ID and not the pool name as is used in other storage systems.
 - Be sure that the `pool` value is the name of an existing pool on the storage system.
-- To create a volume with HyperSwap on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
+- To create a volume with high availability (HA) (HyperSwap or stretched topology) on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
```

Comment:
consider separating the relevant line the line to this:
"HyperSwap on IBM Spectrum Virtualize storage systems, and Stretched topology on SAN Volume Controller"
as stretched topology is supported only on SVC storage systems.

Reply:
I don't think it's necessary, because we indicate this already in the release notes.

Reply:
ok, no problem.